### ISSUE TITLE
IGNN_PPI failed to achieve performance as reported

### ISSUE THREAD

**Issue by XiChen-97** [SwiftieH/IGNN#6]
Hi there @SwiftieH , thank you for your great work on implicit GNN. 

Since I am running the train_IGNN_PPI.py for training IGNN on PPI dataset, I found it's hard to get the micro F1-score, which is around 95% as reported in the paper, instead I get test set F1-score only around 50% after 1000 epochs, the training loss could be larger (over 10,000,000) after 250+ epochs, I think there is something wrong with the training. The experiment setting is all default (lr=0.01, kappa=0.98, no weight decay), the forward and backward iteration in implicit function is 300 respectively. 

I am wondering if there is anything needs to be extra cared, for example increasing the forward/backward iterations to achieve the F1-score results above 95%. Or if I missed some important things in the training, please remind me. Thanks!
